README.md

```diff
@@ -10,40 +10,51 @@ After installation simply comment `/rebase` to trigger the action:
 
 To configure the action simply add the following lines to your `.github/workflows/rebase.yml` workflow file:
 
-```yml
-on: 
+```yaml
+name: Automatic Rebase
+on:
   issue_comment:
     types: [created]
-name: Automatic Rebase
 jobs:
   rebase:
     name: Rebase
     if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/rebase')
     runs-on: ubuntu-latest
     steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          fetch-depth: 0 # otherwise, you will fail to push refs to dest repo
+      - name: Automatic Rebase
+        uses: cirrus-actions/rebase@1.4
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+```
+
+> NOTE: To ensure branch checks run use a [Personal Access Token](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token)
```

Comment:
Maybe add a link to the discussion and clarify that only GitHub Actions won't re-run.

```suggestion
> NOTE: To ensure GitHub Actions will be automatically re-run after a successful rebase action use a workaround of using a [Personal Access Token](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token). See the following [discussion](https://github.community/t/triggering-a-new-workflow-from-another-workflow/16250/37) for more details.
```